src/sprite.rs

```diff
@@ -0,0 +1,182 @@
+use commons::{WIDTH, HEIGHT, Point};
+
+pub fn make_sprite(center: Point, width: f64, height: f64, speed: f64, velocity: [f64; 2]) -> Sprite {
+    Sprite {
+        north_west: Point {
+            x: center.x - width/2.0,
+            y: center.y - height/2.0
+        },
+        south_east: Point {
+            x: center.x + width/2.0,
+            y: center.y + height/2.0
+        },
+        width: width,
+        height: height,
+        velocity: velocity,
+        speed: speed
+    }
+}
+
+pub struct Sprite {
+    north_west: Point,
```

Comment:
We are using cartesian coordinates, maybe "upper-left" and "lower-right" would be better?